Sources/Mantis/CropView/CropScrollView.swift

```diff
@@ -104,10 +107,14 @@ class CropScrollView: UIScrollView {
     
     func resetBy(rect: CGRect) {
         // Reseting zoom need to be before resetting frame and contentsize
-        minimumZoomScale = 1.0
-        zoomScale = 1.0
-
+        minimumZoomScale = max(1.0, initialMinimumZoomScale)
```

Comment:
Let's document that `minimumZoomScale` needs to be `>= 1.0`?

Reply:
Added a comment for it